logutil/scope.go

```diff
@@ -60,7 +60,14 @@ func (s *Scope) Log(log *slog.Logger, attrs ...slog.Attr) *slog.Logger {
 	}
 }
 
-// Err returns an error that propagates the scope's current attributes, plus additional attributes.
+// New creates a new error with the given message and attributes.
+// The error inherits the scope's current attributes.
+func (s *Scope) New(msg string, attrs ...slog.Attr) error {
+	return &scopedError{err: errors.New(msg), msg: msg, group: s.group, attrs: s.concat(attrs)}
```

Comment:
Setting both `msg` and `err` will cause `String()` to output it twice

Reply:
Oops, thank you, will fix